packages/cozy-mjml/src/components/index.js

```diff
@@ -0,0 +1,7 @@
+const core = require('mjml-core')
+
+const components = [require('./MJCozyTitle')]
+
+components.forEach(component => {
+  core.registerComponent(component)
+})
```

Comment:
I am not a fan of side-effects in index.js. I understand the usecase here but I think I would prefer to be explicit and export a `register` function.